src/session/content/chat_action_bar.rs

```diff
@@ -270,6 +290,27 @@ impl ChatActionBar {
         }
     }
 
+    pub(crate) fn handle_paste_action(&self) {
+        if let Some(chat) = self.chat() {
+            let display = gdk::Display::default().unwrap();
+            let clipboard = display.clipboard();
+            clipboard.read_texture_async(
+                gio::Cancellable::NONE,
+                clone!(@weak self as obj => move |r| if let Ok(Some(texture)) = r {
+                    let parent_window = obj.root().unwrap().downcast().ok();
+                    let path = glib::tmp_dir()
```

Comment:
While this works well in a flatpak environment, because flatpak automatically clears this directory on app exit, I don't know if this behaves correctly in an host environment.

Reply:
As I thought, the directory doesn't get deleted there. Also, we should probably create a subdirectory in there so all the data we want to put in /tmp will not be mixed with the rest of the files.

I'm wondering if it's worth just manually deleting that directory on exit at this point.

Reply:
> I'm wondering if it's worth/we can just manually delete that directory on exit at this point.

The answer is yes, that's also what epiphany does.